src/modules/crewMethods.ts

```diff
@@ -58,4 +59,17 @@ export const crewMethods = {
         });
         return getDataFromResponse<Vacancy>(response);
     },
+
+    getGroupList: async (data: GetGroupList) => {
+        const { search, take, filter } = data;
+        const { url, apiToken } = checkConfig();
+        const response = await fetch(`${url}/api/rest/groups/list`, {
+            method: 'POST',
+            headers: {
+                authorization: apiToken,
```

Comment:
Does it work without Content-Type header?

Reply:
Yes, it works